src/main/java/Duke.java

```diff
@@ -1,10 +1,236 @@
-public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
-    }
-}
+import duke.tasks.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+
+public class Duke {
+
+    private static Scanner in = new Scanner(System.in);
+    static final int MAX_SIZE = 100;
+    private static String content = "";
+
+    private static String lineInput = "";
+    private static ArrayList<Task> taskList = new ArrayList<Task>();
+    private static int tasksCount = 0;
+
+    public static void addTask(Task t){
+        if (t.getDescription().length() > 0) {

```

Comment:
It may be possible to introduce a get description length method to avoid this double method calls.